cmd/boulder-va/main.go

```diff
@@ -15,10 +15,16 @@ import (
 	vapb "github.com/letsencrypt/boulder/va/proto"
 )
 
+type RemoteVAConfig struct {
+	cmd.GRPCClientConfig
+	Perspective string `validate:"omitempty"`
+	RIR         string `validate:"omitempty,oneof=ARIN RIPE APNIC LACNIC AfriNIC"`
+}
+
```

Comment:
I previously considered this approach but deemed it too risky. The remote VA backends are configured using SRV records from an in-DC Consul, populated by cloud automation. If the primary VA is set to treat certain backends as "perspective A," but their DNS actually points to "perspective B" backends, it could take a long time to discover the mismatch. The resulting revocation event would include every certificate using an authorization validated by "perspective A".

Instead, we should rely on the remote VAs to report their perspective as part of the ValidationResult. The result of a DNS misconfiguration or a primary VA configured to  to lookup the wrong SRV record would be an increase in validations which now fail because they cannot reach quorum.

Reply:
Ack, I was working on a separate change and accidentally pushed it as part of this branch! Sorry for the confusion. I've force-pushed to remove that last erroneous commit.